lib/jekyll/filters.rb

```diff
@@ -273,7 +273,7 @@ def to_integer(input)
       begin
         input.to_i
       rescue
-        raise ArgumentError, "Invalid input object type."
+        raise ArgumentError, "Object '#{input.inspect}' could not be converted into an integer."
```

Comment:
Thanks. It should be solved.
I learned the lesson that ```script/fmt``` should be run before commits. 😄